Dev4Agriculture.ISO11783.ISOXML/ISOExtensions/ISOPartField.cs

```diff
@@ -63,7 +63,32 @@ public bool IsInField(decimal longitude,decimal latitude)
         {
             return PolygonnonTreatmentZoneonly.First().IsInPolygon(longitude, latitude);
         }
-        
+
+        public bool IsIntersectWithField(ISOPartfield secondField, out double intersectPercent)
```

Comment:
For naming consistancy, please name this TryGetOverlapWithPartfield or TryGetIntersectionWithPartfield.

Specifically TryGet instead of Is to mark that we have a get (Out) and a Try => Bool result